include/disccord/rest/api_client.hpp

```diff
@@ -224,6 +225,23 @@ namespace disccord
 
                     pplx::task<void> request(route_info& route, const pplx::cancellation_token& token = pplx::cancellation_token::none());
 
+                    pplx::task<web::json::value> request_raw_json(route_info& route, const pplx::cancellation_token& token = pplx::cancellation_token::none());
```

Comment:
IMO if you need the response body not parsed as a model you should use the internal request methods